### PR TITLE
Misc CSS cleanup

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

- improve button hover state everywhere
- consistent transitions
- fix mobile menu in darkmode
- remove feedback button on mobile
- consolidate menu styles into single css file

Fixes https://github.com/jbx-protocol/juice-interface/issues/1936

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/ManageTokensModal.tsx [hunk 1]
```diff
@@ -10,20 +9,8 @@ import { veNftPagePath } from 'utils/routes'
 import { tokenSymbolText } from 'utils/tokenSymbolText'
 import { reloadWindow } from 'utils/windowUtils'
 
-const BURN_DEFINITION_LINK =
-  'https://www.investopedia.com/tech/cryptocurrency-burning-can-it-manage-inflation/'
-
 type RedeemDisabledReason = 'redemptionRateZero' | 'overflowZero'
 
-const BurnTokensHelp = () => {
```
Comment (tomquirk) [inline src/components/ManageTokensModal.tsx:18]: Unnecessary tooltip

File: src/components/ManageTokensModal.tsx [hunk 2]
```diff
@@ -193,18 +176,20 @@ export default function ManageTokensModal({
...
-                  <Trans>
-                    Mint new {tokensLabel} into an account. Only a project's
-                    owner, a designated operator, or one of its terminal's
-                    delegates can mint its tokens.
-                  </Trans>
-                }
-                onClick={() => setMintModalVisible(true)}
-                disabled={!projectAllowsMint}
-              />
+              <div>
```
Comment (tomquirk) [inline src/components/ManageTokensModal.tsx:179]: Need a nesting div so the Tooltip doesn't override the `cliable-button` class

File: src/styles/antd-overrides/layout.scss [hunk 3]
```diff
@@ -1,12 +1,3 @@
-.ant-menu,
```
Comment (tomquirk) [inline src/styles/antd-overrides/layout.scss:1]: move to `menu.scss`

File: src/styles/antd-overrides/menu-item.scss [hunk 4]
```diff
@@ -1,25 +0,0 @@
-.ant-menu-item:hover {
```
Comment (tomquirk) [inline src/styles/antd-overrides/menu-item.scss:1]: move to `menu.scss`

File: src/styles/antd-overrides/button.scss [hunk 5]
```diff
@@ -21,14 +22,14 @@
 
 .ant-btn-primary {
   &:not(.ant-btn[disabled]) {
-    background: var(--background-action-primary) !important;
-    border-color: transparent !important;
-    color: var(--text-over-action-primary) !important;
+    background: var(--background-action-primary);
+    border-color: transparent;
+    color: var(--text-over-action-primary);
```
Comment (tomquirk) [inline src/styles/antd-overrides/button.scss:27]: important not needed, and was overriding the hover state

File: src/styles/index.scss [hunk 6]
```diff
@@ -57,10 +57,10 @@ h6 {
 
 .clickable-border {
   border: 1px solid var(--stroke-tertiary);
-  transition: border-color 120ms ease-out;
+  transition: border-color 100ms linear;
 
   &:hover {
-    border: 1px solid var(--stroke-secondary);
+    border: 1px solid var(--stroke-primary);
```
Comment (tomquirk) [inline src/styles/index.scss:63]: stronger card hover state

File: src/pages/create/forms/FundingForm/DistributionSplitsSection/DistributionSplitCard.tsx [hunk 7]
```diff
@@ -31,6 +31,10 @@ const Parens = ({
   return <>{children}</>
 }
 
+const gutter = 10
+const labelColSpan = 9
+const dataColSpan = 15
```
Comment (tomquirk) [inline src/pages/create/forms/FundingForm/DistributionSplitsSection/DistributionSplitCard.tsx:36]: same, just moved out of component scope

**Review by wraeth-eth — APPROVED** [review]

**Pull request merged**